src/factions/Tyranids/models.json

```diff
@@ -870,6 +870,10 @@
           {
             "WeaponId": "DEV",
             "Cost": 0
+          },
+          {
+            "WeaponId": "CRM",
+            "Cost": 2

```

Comment:
This should probably be modeled as the base cost and the other ones as the override (like it shows in the annual), but this is "good enough" for now to get the points in the right spot.

Reply:
Yeah that's the the thing, it was the base cost and the Warrior wasn't picking it up! Hence the override.

Reply:
https://github.com/KTManager/KTRules/blob/2c719e1e024438379b1bf524484d16ef342f6f6a/src/weapons.jsonnet#L2589